runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -167,12 +167,12 @@ TR_J9ServerVM::isMethodEnterTracingEnabled(TR_OpaqueMethodBlock *method)
       auto it = _compInfoPT->getClientData()->getJ9MethodMap().find((J9Method*) method);
       if (it != _compInfoPT->getClientData()->getJ9MethodMap().end())
          {
-         return it->second._isMethodEnterTracingEnabled;
+         return it->second._isMethodTracingEnabled;
```

Comment:
`isMethodEnterTracingEnabled` and `isMethodExitTracingEnabled` could be deleted and `isMethodTracingEnabled` could be overridden following this pattern. Thus, when somebody calls `isMethodEnterTracingEnabled` the J9Base implementation will be called which in turn calls the virtual implementation of `isMethodTracingEnabled` which knows how to get the information from the client.